src/app/network/network-list/network-card/network-card.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="mb-4 w-full lg:flex c-network-card" data-testid="c-network-card">
+	<div
+		class="w-full border border-gray-400 bg-white rounded-lg leading-normal"
```

Comment:
Do not use hardcoded colors from Tailwind, you should get them from css variables, as we also offer dark mode.